help/help.md

```diff
@@ -0,0 +1,37 @@
+% POSTFIX(1) Container Image Pages
```

Comment:
Can we please move usage from README to help page and just link it from readme?

Reply:
yes

Reply:
my point is: help page needs to have info which is currently placed in readme: so we can
1. template
2. link
3. duplicate